plugins/generic/pln/classes/DepositPackage.inc.php

```diff
@@ -111,19 +132,19 @@ function generateAtomDocument() {
 		$entry->setAttributeNS('http://www.w3.org/2000/xmlns/' ,'xmlns:dcterms', 'http://purl.org/dc/terms/');
 		$entry->setAttributeNS('http://www.w3.org/2000/xmlns/' ,'xmlns:pkp', 'http://pkp.sfu.ca/SWORD');
 		
-		$email = $atom->createElement('email', $journal->getSetting('contactEmail'));
+		$email = $this->_generateElement($atom, 'email', $journal->getSetting('contactEmail'));
```

Comment:
I figure it's possible that a journal could have bad byte sequences or enter the email as 
`"Bob Terwilliger" <bob@example.com>`

Best to be safe.